lib/usdUfe/python/wrapCommands.cpp

```diff
@@ -104,6 +104,9 @@ void wrapCommands()
         class_<This, PXR_BOOST_PYTHON_NAMESPACE::noncopyable>("ClearDefaultPrimCommand", no_init)
             .def("__init__", make_constructor(ClearDefaultPrimCommandInit))
             .def("execute", &UsdUfe::UsdUndoClearDefaultPrimCommand::execute)
+#ifdef UFE_V4_FEATURES_AVAILABLE
+            .def("commandString", &UsdUfe::UsdUndoClearDefaultPrimCommand::commandString)
+#endif
```

Comment:
Added missing commandString python binding for all these Ufe commands. Its a shame they don't inherit the bindings from the PyBind11 ones in the base class from Ufe.